module/zfs/.dmu_zfetch.o.tmp

```diff
@@ -0,0 +1,1300 @@
+cmd_/media/sf_projects/zfs/module/zfs/dmu_zfetch.o := gcc -Wp,-MD,/media/sf_projects/zfs/module/zfs/.dmu_zfetch.o.d  -nostdinc -isystem /usr/lib/gcc/x86_64-linux-gnu/5/include  -I./arch/x86/include -I./arch/x86/include/generated  -I./include -I./arch/x86/include/uapi -I./arch/x86/include/generated/uapi -I./include/uapi -I./include/generated/uapi -include ./include/linux/kconfig.h -Iubuntu/include  -D__KERNEL__ -Wall -Wundef -Wstrict-prototypes -Wno-trigraphs -fno-strict-aliasing -fno-common -fshort-wchar -Werror-implicit-function-declaration -Wno-format-security -std=gnu89 -fno-PIE -mno-sse -mno-mmx -mno-sse2 -mno-3dnow -mno-avx -m64 -falign-jumps=1 -falign-loops=1 -mno-80387 -mno-fp-ret-in-387 -mpreferred-stack-boundary=3 -mskip-rax-setup -mtune=generic -mno-red-zone -mcmodel=kernel -funit-at-a-time -DCONFIG_X86_X32_ABI -DCONFIG_AS_CFI=1 -DCONFIG_AS_CFI_SIGNAL_FRAME=1 -DCONFIG_AS_CFI_SECTIONS=1 -DCONFIG_AS_FXSAVEQ=1 -DCONFIG_AS_SSSE3=1 -DCONFIG_AS_CRC32=1 -DCONFIG_AS_AVX=1 -DCONFIG_AS_AVX2=1 -DCONFIG_AS_AVX512=1 -DCONFIG_AS_SHA1_NI=1 -DCONFIG_AS_SHA256_NI=1 -pipe -Wno-sign-compare -fno-asynchronous-unwind-tables -mindirect-branch=thunk-extern -mindirect-branch-register -DRETPOLINE -fno-delete-null-pointer-checks -O2 --param=allow-store-data-races=0 -DCC_HAVE_ASM_GOTO -Wframe-larger-than=1024 -fstack-protector-strong -Wno-unused-but-set-variable -fno-omit-frame-pointer -fno-optimize-sibling-calls -fno-var-tracking-assignments -g -pg -mfentry -DCC_USING_FENTRY -Wdeclaration-after-statement -Wno-pointer-sign -fno-strict-overflow -fno-merge-all-constants -fmerge-constants -fno-stack-check -fconserve-stack -Werror=implicit-int -Werror=strict-prototypes -Werror=date-time -Werror=incompatible-pointer-types -Werror=designated-init -std=gnu99 -Wno-declaration-after-statement -Werror -fno-inline -include /media/sf_projects/zfs/zfs_config.h -I/media/sf_projects/zfs/include/spl -I/media/sf_projects/zfs/include -std=gnu99 -Wno-declaration-after-statement -Werror -fno-inline -include /media/sf_projects/zfs/zfs_config.h -I/media/sf_projects/zfs/include/spl -I/media/sf_projects/zfs/include -D_KERNEL -DDEBUG -UNDEBUG -DDEBUG_KMEM -DDEBUG_KMEM_TRACKING -D_KERNEL -DDEBUG -UNDEBUG -DDEBUG_KMEM -DDEBUG_KMEM_TRACKING  -DMODULE  -DKBUILD_BASENAME='"dmu_zfetch"'  -DKBUILD_MODNAME='"zfs"' -c -o /media/sf_projects/zfs/module/zfs/dmu_zfetch.o /media/sf_projects/zfs/module/zfs/dmu_zfetch.c
```

Comment:
this file shouldn't be included.

Reply:
no idea how git didnt ignore that. but ok..